src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -939,6 +928,47 @@ def body(self):
         finally:
             os.remove(config_file)
 
+class VMSSExtensionInstallTest(VCRTestBase):
```

Comment:
these follow the pattern of the existing extension tests, which we can update all together if we need to.  I expect these tests, like the vm extension tests, to not be re-recorded often

---
In reply to: [76078126](https://github.com/Azure/azure-cli/pull/737#discussion_r76078126) [](ancestors = 76078126)